spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -88,6 +89,7 @@
 
   scenario "User publishes a page" do
     given_there_is_a_draft_step_by_step_page
+    and_I_have_checked_for_broken_links
```

Comment:
This and the previous step can be combined to use `given_there_is_a_step_by_step_page_with_a_link_report`